Dockerfile

```diff
@@ -8,25 +8,22 @@
 #  docker run -it  pmoxs3backuproxy:latest pmoxs3backuproxy -endpoint 127.0.0.1:9000
 #
 # Note: uses default server key and certificate stored in /root/
-FROM debian:bookworm-slim
+FROM golang:1.22.5 as build
```

Comment:
should be `FROM .... AS build` (note uppercase)

Reply:
works for me with Docker version 20.10.24+dfsg1, build 297e128 nevertheless.

Reply:
I think it still works but it produces a build-time error

Reply:
pushed 024ca29